overview/dwavesys.rst

```diff
@@ -55,21 +55,20 @@ Client tool installation).
 .. code-block:: bash
 
     $ dwave config create
-    Configuration file not found; the default location is: C:\\Users\\jane\\AppData\\Local\\dwavesystem\\dwave\\dwave.conf
-    Confirm configuration file path (editable):
-    Profile (create new): prod
-    API endpoint URL (editable): https://my.dwavesys.url/
-    Authentication token (editable): ABC-1234567890abcdef1234567890abcdef
-    Client class (qpu or sw): qpu
-    Solver (can be left blank): {"qpu": true, "vfyc": true}
-    Proxy URL (can be left blank):
+    Configuration file not found; the default location is: /home/jane/.config/dwave/dwave.conf
+    Confirm configuration file path [/home/jane/.config/dwave/dwave.conf]:
+    Profile (create new) [prod]:
+    API endpoint URL [skip]:
+    Authentication token [skip]: ABC-1234567890abcdef1234567890abcdef
+    Default client class (qpu or sw) [qpu]:
+    Default solver [skip]:
     Configuration saved.
 
-2. Enter the SAPI information (e.g. your API token) found as described above. You can
-   accept the command's defaults and in the future update the file if needed.
-
-   To get started, Leap users can create a minimum configuration by entering only an API
-   token; on-premises users should also set the URL to the on-premises system.
+2. Enter the SAPI information (e.g. your API token) found as described in the section
+   above. To get started, create a minimum configuration by accepting the command's
+   defaults (pressing Enter) for all prompts except the API token (Leap users) or
+   API token and endpoint (on-premises users). You can in the future update the
+   file if needed.
```

Comment:
... by either running the same command, and selecting the same profile, or by manually editing the config file.

Reply:
@randomir, my preference for this item was to provide minimum detail -- this is a Getting Started step for first-time users, and "You can in the future..." a sentence of guidance for a potential future change. I weigh keeping the Getting Started steps short to encourage users to actually read & follow them over a wall of comprehensive text they will see, groan, and close. Especially, when it comes to a potential future change, which they are unlikely to remember from reading it here, and unlikely to be reading here when the time comes to make that change.

Reply:
Ok. I thought it would be useful to mention the fact they can edit existing configuration by running the `dwave config create` command again, especially since you're already talking here how to change the config _in the future_ -- by editing the file (but they don't need to manually edit files). Alternatively, it would be nice to put this our CLI docs, once we have them.

Re: wall of text, may I suggest we:
- only emphasize the need for API token (in "Interacting with SAPI"), and mention endpoint and solver have default values. Cuts the first section in half.
- remove the `dwave sample` example. Adds no value over ping. It should go to CLI docs.
- remove "properties" examples (both CLI and code) from getting started. Instead maybe just list solvers with `dwave solvers -l` and mention more details available with `dwave solvers`?

Also, instead of using "My_DWAVE..." I would keep the output accurate. Same for "http://my.dwavesys.url". Oh, and I think it's important to note that `get_solvers` example works standalone, *without any configuration*, because token is the only info needed to run a default Leap solver.

Reply:
Thanks for the suggestions. I put this in https://github.com/dwavesystems/docs/issues/71. 
For this PR I just want to clarify, re " since you're already talking here how to change the config in the future -- by editing the file", that the current phrasing is "You can in the future update the file if needed", which includes updating by rerunning dwave CLI rather than just manually editing. 

Also worth keeping in mind for the last paragraph that these instructions also apply to on-premises users, so places where we have "http://my.dwavesys.url" are meant for them.

Reply:
Yes, "update the file" technically includes both scenarios, although (in my head):
- I update the *file* by editing it
- I update the *configuration* by running a configuration tool (which might write to the same file)

Re: on-prem users. Do you think describing a special case (for a minority of users) in the entry-level getting started guide warrants the distraction it's causing for the majority of Leap users?

Reply:
On-premises here was a request from @murraythom. I'm thinking that in the next revision I'd want a separate section for on-premises users with also some additional info such as security certificates, etc

Reply:
I agree, @JoelPasvolsky, separate section, or even page, makes sense.